.swiftlint.yml

```diff
@@ -1,4 +1,6 @@
 excluded:
+  - Carthage
+  - Pods
   - third_party
 
 disabled_rules:
```

Comment:
Let's start with no disabled rules—you can run `swiftlint autocorrect` on the command line to auto-fix everything. If we find some rules are really egregious then we can adjust in the future.